uw-research-computing/gpu-jobs.md

```diff
@@ -217,6 +221,13 @@ attributes shown above. If you want a certain class of GPU, use CUDACapability:
 	code so that it can run across GPU types and without needing the
 	latest version of CUDA.
 
+- **Specify Multiple Requirements (optional)**: Multiple requirements can be specified by using && statements:
+	```
+	requirements = (CUDACapability == 7.5) && (CUDADeviceName = "GeForce RTX 2080 Ti")
+	```
+	{:.sub}
+	Ensure all specified requirements match the attributes of the machine in interest. HTCondor matches jobs to gpus that match all specified requirements. Otherwise, the jobs will sit idle indefinitely
```

Comment:
```suggestion
	Ensure all specified requirements match the attributes of the GPU/server of interest. HTCondor matches jobs to GPUs that match all specified requirements. Otherwise, the jobs will sit idle indefinitely.
```